src/static/riot/competitions/editor/_competition_details.tag

```diff
@@ -71,30 +62,23 @@
                 })
             })
 
-            $('.queue-search')
-                .dropdown({
-                    apiSettings: {
-                        url: `${URLS.API}queues/?search={query}`,
-                    },
-                    clearable: true,
-                    preserveHTML: false,
-                    minCharacters: 2,
-                    fields: {
-                        remoteValues: 'results',
-                        name: 'name',
-                        value: 'id',
-                    },
-                    cache: false,
-                    //cache: false,
-                    maxResults: 5,
-                    onChange: (value, title) => {
-                        self.refs.queue.value = value
-                        self.form_updated()
-                    }
-                })
-            ;
-
-            self.get_available_queues()
+            $(self.refs.queue).dropdown({
+                // Note: Passing `public=true` so default behavior is users can search for public queues
```

Comment:
Does passing `public=True` screw up the `get_queryset` at all? since we list 'public' as one of the filter fields in the `QueueViewSet`, will this filter out all the user's queues that are not set to public? I can't remember the firing order of 'filter_backends' and 'get_queryset'

Reply:
This is something I'll have to verify. The behavior when tested previously was working as I wanted.

Reply:
![image](https://user-images.githubusercontent.com/28552312/68415103-7164ee80-0146-11ea-900b-11e48127a527.png)

Talked to you previously on Slack, I believe this verifies that user's queues that are not public will not be filtered out.

Reply:
Also added a test for this